packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -187,6 +189,18 @@ export default declare((api, options) => {
     },
   };
 
+  const privateMethodHandlerSpec = {
+    ...privateNameHandlerSpec,
```

Comment:
Looks like this will inherit the behavior to use classPrivateFieldSet. I don't think that's what you want here; you probably want to throw a TypeError for methods and call the setter for accessors.